### ISSUE TITLE
User interface fixes

### ISSUE THREAD

**Issue by qtrandev** [CodeforSouth/LeafletTransit#33]
Make any user interface fixes you think is necessary. You now have direct write access. Only commit to "dev" branch - I'll pull into master and gh-pages.

**Comment by bruncun** [comment]
What do you think of collapsing all "show" options into a single multiselect? I originally got the idea from the Apple Maps Desktop client. We could use this bootstrap-multiselect plugin - http://davidstutz.github.io/bootstrap-multiselect/, and rewire the event handlers from on click to checking the different options. This would maximize our screen space on mobile and clean up our design some more.
![image](https://cloud.githubusercontent.com/assets/6834144/11842493/f277a8f4-a3d1-11e5-8898-8224b8c8d673.png)

**Comment by qtrandev** [comment]
I don't like using drop-downs too much since it doesn't make the interface intuitive and makes the function a 2-action process. Also, the more plugins we had, the harder to maintain this code since now I would need to learn how that plugin works. Either way, we might need to embrace change to improve. If you want to try it out, put it on a branch in your project and we can take a look.

**Comment by bruncun** [comment]
Hm, I just noticed that the menu beneath map controls already contains a dropdown where users can toggle different features of the map. @qtrandev, were the map controls intended to be any different from this feature? Seems like a potential redundancy.

**Comment by qtrandev** [comment]
I have the big buttons because I wanted it to be in-the-face of the user to make it dead-easy to use. The stack-of-paper icon to show more controls is how Leaflet allows layers to be toggled - I don't like it too much, but it's useful right now since we have lots of layers.

**Comment by bruncun** [comment]
Hmm, I also noticed this morning the mock ups that were posted. Should we consider adding another navbar, fixed to the bottom of the window, with icon buttons for toggling each feature?

**Comment by qtrandev** [comment]
Yeah, a bottom navbar might be useful - paste a mockup picture or just implement in a branch if you'd like.

**Comment by bruncun** [comment]
Ok, I can start experimenting with moving controls to double navbars so we can maximize in the face controls while still freeing space inside the map pane to improve experience.
